client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -117,6 +124,8 @@ const RoundManagement = ({ round, auditBoards, createAuditBoards }: IProps) => {
                 <QRs passphrases={auditBoards.map(b => b.passphrase)} />
                 <RoundProgress auditBoards={auditBoards} round={round} />
               </>
+            ) : auditType === 'BATCH_COMPARISON' ? (
+              <BatchRoundDataEntry round={round} />
```

Comment:
this conditional seems a bit sketchy cuz it relies on `online` always being `false` for batch audits. maybe just move this case to the beginning of the condition, so that we only test `online` once we know we're not in a batch audit

Reply:
@jonahkagan We hard code it to `online: false` if it's in a batch audit, though, so that's a given, right?

Reply:
yes, but reading this bit of code I didn't know that, and there's no other code that I know of that makes that assumption so if that ever changed this would just quietly break. it's only hard-coded to false because the default value in the db is already set to false - I don't think it was an intentional decision

Reply:
@jonahkagan Okay, had to look to find it, haha, but here's the original decision about this: https://github.com/votingworks/arlo/issues/639

Batch comparison audits are designed to only support offline. The tests are a bit sparse around this section, so I think we should include some tests to verify that when it's a batch audit it automatically submits it as offline during the settings stage. I seriously thought I'd already covered that test case somewhere but I can't find it anywhere, which is annoying. In the meantime I've added comments to clarify what's going on.